File: Angular2PoC/RoutingAndNavigation/app/log/entry.model.ts
import {DateTimeModel} from './datetime.model';
//import {ActionDirection} from './actiondirection.model.ts';

export class EntryModel {

    constructor(public logEntryId: number, public name: string, public time: DateTimeModel, public direction: number) { }

    

    getTime() {
        return "11:11"
        //return ("0" + (this.date.getHours())).slice(-2) + ":" + ("0" + (this.date.getMinutes())).slice(-2);//+ ":" + ("0" + (this.date.getSeconds())).slice(-2);
    }
}

File: OfficeHoursXamarin/OfficeHours/OfficeHours/Models/EntryModel.cs
using System;

namespace OfficeHours
{
	public class EntryModel
	{
		public int LogEntryId { get; set; }
		public DateTime Time { get; set; }
		public ActionDirection Direction { get; set; }
		public string Name { get; set; }


		public EntryModel ()
		{
		}
	}
}

